pkg/generator_test.go

```diff
@@ -2306,6 +2305,14 @@ func NewA(t mockConstructorTestingTNewA) *A {
 	s.checkGeneration("struct_value.go", "A", false, "", expected)
 }
 
+func (s *GeneratorSuite) TestGeneratorForStructWithTag() {
+	b, err := os.ReadFile("testdata/struct_with_tag.txt")
+	s.NoError(err)
+
+	expected := string(b)
+	s.checkGeneration("struct_with_tag.go", "StructWithTag", false, "", expected)
```

Comment:
Thanks review!
I fixed in this commit: https://github.com/vektra/mockery/pull/569/commits/dd639a33c2a928c151d31dbfb237a27e30a1a7c7

Because expected string is enough short, that removed `testdata/struct_with_tag.txt`.